src/components/ResultDisplay.tsx

```diff
@@ -69,3 +72,14 @@ export function ResultDisplay({ result }: ResultDisplayProps) {
 function counted(count: number, text: string) {
 	return `${count} ${text}${count === 1 ? "" : "s"}`;
 }
+
+function summed(result: EstimatedPackage[]) {
```

Comment:
[Suggestion] Not a blocker but maaaaybe a more precise name like `estimations`/`packages` would be easier to reason about for this function?

Reply:
I agree, I think `sumEstimateFunding` makes it clear what this function is doing

Reply:
Oh I meant for the function parameter... but yes the function name too. Both!

Reply:
por que no los dos